src/countOccurrences.js

```diff
@@ -19,6 +19,15 @@
  */
 function countOccurrences(phrase, letter) {
   // write code here
+  const result = [];
+  const arrPhrase = phrase.toLowerCase().split('');
+  const lowerLetter = letter.toLowerCase();
+  for (let i = 0; i < arrPhrase.length; i++) {
+    if (arrPhrase[i].includes(lowerLetter)) {
+      result.push(arrPhrase[i]);
+    }
+  }
```

Comment:
new line before return
```suggestion
  }
  
```